src/main.ts

```diff
@@ -10,16 +12,22 @@ export class Application {
 	 * @returns {Promise<void>} Returns just a promise with now result.
 	 */
 	public async main (): Promise<void> {
+		const environment: Environment = new Environment();
 		const actionInput: Action = new Action();
-		
+		const tagAPI: TagAPI = new TagAPI();
+
 		try {
-			// Refer to the action.yml file for the list of inputs setup for the action
-			const inputValue: string = actionInput.getInput("sample-input-name");
+			const tagName: string = environment.getVarValue("tag-name");
+			const tagExists: boolean = await tagAPI.tagExistsAsync("v1.0.0-preview.5");
```

Comment:
Forgot to remove magic string during dev/testing.

```suggestion
			const tagExists: boolean = await tagAPI.tagExistsAsync(tagName);
```